src/cobra/core/reaction.py

```diff
@@ -538,53 +715,97 @@ def functional(self):
         return True
 
     @property
-    def x(self):
-        """The flux through the reaction in the most recent solution.
+    def x(self) -> float:
```

Comment:
These have been deprecated for some time. Maybe it's time to remove them?

Reply:
Maybe a separate PR to get rid of deprecations?